frontend/src/components/common/button/button.tsx

```diff
@@ -63,6 +63,7 @@ const Button: FC<Props> = ({
       <Link
         to={to as AppRoute}
         className={getValidClasses(
+          className,
```

Comment:
Remove className property from Button component, it should have 100% width, and we should limit it via wrappers where it's needed. If we need somewhere more colors or another size, then add new property for button

Reply:
agreed